src/styles/fixedProgress.styl

```diff
@@ -0,0 +1,25 @@
+progress
+    width         100%
+    margin        1em 0
+    color         dodger-blue
+    font-size     .6em
+    line-height   1.5em
+    text-indent   .5em
+    height        1em
+    background    grey-01
+    border        0
+    border-radius 20px
+
+    // chrome and safari
+    &::-webkit-progress-bar
+        background    grey-01
+        border-radius 20px
+
+    &::-webkit-progress-value
+        background    blue
```

Comment:
blue? that's not a color!  (well yes obviously it is, but not a color from the styleguide) Is there a reason?